assignments/callbacks.js

```diff
@@ -24,28 +24,52 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, function(length) {
+  console.log(length);
+})
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length - 1]);
 }
 
+last(items, function(lastItem) {
+  console.log(lastItem)
+})
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y)
 }
 
+sumNums(3, 4, function(result) {
+  console.log(result)
+})
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
 }
 
+multiplyNums(3, 5, function (result) {
+  console.log(result);
+})
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  return cb(list.include(item))
 }
 
+contains('Gum', items, function(result) {
+  console.log(result);
+})
+
```

Comment:
function contains(item, list, cb) {
  // contains checks if an item is present inside of the given array/list.
  // Pass true to the callback if it is, otherwise pass false.
  for (let i = 0; i < list.length; i++) {
    if (list[i] === item) {
      return cb(true);
    }
  }
  return cb(false);
}

contains('Bats', items, function(flag) {
  console.log(flag); // should be false
});

contains('Notebook', items, function(flag) {
  console.log(flag); // should be true
});